### DISCUSSION TITLE
Conflicting info on Codespaces availability for free accounts

### DISCUSSION THREAD

**Discussion by diminutivesloop** [community/community#57364] [Codespaces]
### Select Topic Area

Product Feedback

### Body

On the [billing page](https://github.com/settings/billing) the description of what's included with my free plan lists "Access to Codespaces" as "not included" while claiming I have access to "120 core-hours of Codespaces compute" and "15GB of Codespaces storage". 🤷 

<img width="787" alt="Screenshot 2023-06-06 at 2 32 20 PM" src="https://github.com/community/community/assets/667716/42a8bc2c-bceb-4c66-9ca1-ed31af0808ee">

**Comment by spenserblack** [comment]
Yeah, displaying these as a binary can be a bit confusing, but I think the meaning is that you can use codespaces, but you don't have access to the *extra* codespace features (more core hours, more storage).

Additionally, a lot of these "not included" features are available for *public* repositories, regardless of plan.

Besides "infinite" (infinity symbol), "yes" (check mark), and "no" (X) icons, perhaps GitHub can make this clearer by adding symbols to represent "limited" and "public-only," too. Though I have no idea what those icons would be :laughing: :shrug:

**Comment by dmilburn** [comment]
Hi @diminutivesloop, thanks for calling out! Agreed this wording is confusing. We have an internal issue working to clear up this confusion on that page. Here's what that was trying to express:

- Free GitHub user accounts have access to 120 core hours of free Codespaces usage. After that, free users can choose to pay for additional Codespaces use.
- Free organizations are unable to pay for Codespaces usage on their repositories. Paid organizations are able to choose whether or not to pay for Codespaces usage on their repositories.

**Reply by spenserblack** [reply]
Thanks for clearing that up!

>Paid organizations are able to choose whether or not to pay for Codespaces usage on their repositories.

Is that limited to usage by organization members?

**Reply by dmilburn** [reply]
No problem. Organization admins for paid organizations can choose between these options under the Codespaces section of organization settings:

<img width="988" alt="Screenshot 2023-06-07 at 5 31 48 PM" src="https://github.com/community/community/assets/10053402/5471934c-0705-41cc-9a99-23e7bf35fddf">


This can be found at https://github.com/organizations/<org_name>/settings/codespaces
